test/integration/047_dbt_ls_test/test_ls.py

```diff
@@ -94,6 +94,8 @@ def expect_snapshot_output(self):
                     'alias': None,
                     'check_cols': None,
                 },
+                'unique_id': 'snapshot.test.my_snapshot',
+                'original_file_path': 'snapshots/snapshot.sql',
```

Comment:
Looks like this is raising an error on Windows because paths are different. I think we have [`normalize()`](https://github.com/fishtown-analytics/dbt/blob/0f018ea5dd678d37668c819552328b3254c2f220/test/integration/base.py#L32-L42) + [`Normalized()`](https://github.com/fishtown-analytics/dbt/blob/0f018ea5dd678d37668c819552328b3254c2f220/test/integration/base.py#L45-L56) for this purpose.

Reply:
Done!